src/common_utils/tests/mock/sample_config_for_sql.json

```diff
@@ -0,0 +1,25 @@
+{
+  "unique_id": "unique_id_to_filter_bq_result_table",
+  "batchDistribution": 1,
+  "type": "sql",
+  "source": "teradata",
+  "extract_ddl": "no",
+  "validation_config": {
```

Comment:
Isn't it required to provide SQL translation details?

Reply:
Probably added the way for Edge case Testing